### ISSUE TITLE
Feature: Automatically Detect if user is trying to send a url

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#909]
Some users are having a hard time differentiating between a DataURL and a URL. This leads to people trying to set the file param as a url. Although it's wrong, I can't blame them for doing what is intuitive.

It may be useful to have some super basic logic that will determine if the file parameter is actually a URL, if it is then download the file and automatically format it properly.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [2.0.30](https://github.com/open-wa/wa-automate-nodejs/compare/2.0.29...2.0.30) (2020-10-10)

- build(deps): bump hasha from 5.2.1 to 5.2.2 [`#913`](https://github.com/open-wa/wa-automate-nodejs/pull/913)
- build(deps-dev): bump @types/node from 14.11.5 to 14.11.8 [`#914`](https://github.com/open-wa/wa-automate-nodejs/pull/914)
- build(deps-dev): bump release-it from 14.0.3 to 14.0.4 [`#908`](https://github.com/open-wa/wa-automate-nodejs/pull/908)
- build(deps-dev): bump @types/node from 14.11.2 to 14.11.5 [`#906`](https://github.com/open-wa/wa-automate-nodejs/pull/906)
- Release 2.0.30 [`be895a8`](https://github.com/open-wa/wa-automate-nodejs/commit/be895a8a3b5e13ec8a8838513684718298bf07a2)
- Implemented #909 [`dea9290`](https://github.com/open-wa/wa-automate-nodejs/commit/dea929069f41558687a04cfda2eeca0482d57d8f)
- Feature: last 4 sent with broken methods for better debugging #893 [`ea4b8db`](https://github.com/open-wa/wa-automate-nodejs/commit/ea4b8db6c9e334b322467c9436a119245551ab22)
- possible fix for #915 [`3caa185`](https://github.com/open-wa/wa-automate-nodejs/commit/3caa185066328203a25af2da10c674a69b8ad497)
